advantageswervesomeworking/src/main/java/frc/robot/subsystems/ShuffleBoard.java

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.subsystems.drive.Drive;
+import frc.robot.subsystems.intake.IntakeSubsystem;
+import frc.robot.subsystems.shooter.ShooterSubsystem;
+
+public class ShuffleBoard extends SubsystemBase {
+  /** Creates a new ShuffleBoard. */
+  IntakeSubsystem intakeSubsystem;
+  ShooterSubsystem shooterSubsystem;
+  Drive drive;
+  public ShuffleBoard(
+    IntakeSubsystem intakeSubsystem,
+    ShooterSubsystem shooterSubsystem,
+    Drive drive) {
+
+      this.intakeSubsystem = intakeSubsystem;
+      this.shooterSubsystem = shooterSubsystem;
+      this.drive = drive;
+  }
+
+  @Override
+  public void periodic() {
+    // This method will be called once per scheduler run
+    SmartDashboard.putBoolean("shuffleboardsubsystem/intake/noteInside", intakeSubsystem.checkForNoteInside());
+    SmartDashboard.putNumber("shuffleboardsubsystem/intake/speed", intakeSubsystem.getIntakePower());
+    SmartDashboard.putNumber("shuffleboardsubsystem/shooter/speed", shooterSubsystem.getShooterPower());
+    SmartDashboard.putString("shuffleboardsubsystem/drive/pose", drive.getPose().toString());
+  }
+}
```

Comment:
A to nie w każdym systemie powinna być metoda z logami, żeby to było oddzielone, chyba że teraz tak się robi to git

Reply:
wlasnie nie wiem, bo w tamtym roku chcialem wszystkie logi w jednej klasie - pliku ale cos mi nie dzialalo, wiec testuje, jak bedzie lepiej. Jak bedzie slabo/ nie bedzie dzialac to na kazdym subsystemie osobno zrobimy.